src/NServiceBus.AmazonSQS/ReducedPayloadContractResolver.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace NServiceBus.AmazonSQS
+{
+    using System.Collections.Generic;
+    using System.Reflection;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Serialization;
+    class ReducedPayloadContractResolver : DefaultContractResolver
+    {
+        static List<string> PropertiesToIgnore = new List<string>
```

Comment:
Since we’re not touching it, is this better as a `string[]` instead of a `List<>`. Nitpicking, probably.